packages/generator/package.json

```diff
@@ -51,17 +51,13 @@
     "build:ts:swc": "swc src --out-dir build --strip-leading-paths --source-maps",
     "build:ts:tsc": "tsc --emitDeclarationOnly",
     "build:ts:tsc-alias": "tsc-alias",
-    "dev:directus": "dotenv -e directus.env.development directus --",
-    "dev:directus:schema:apply": "dotenv -e directus.env.development directus schema apply ./prisma/directus-snapshot.yml",
+    "dev:directus": "dotenv -e ../../env/development/directus.env.development directus --",
+    "dev:directus:bootstrap": "npm run dev:directus bootstrap",
+    "dev:directus:schema:apply": "dotenv -e ../../env/development/directus.env.development directus schema apply ./prisma/directus-snapshot.yml",
     "dev:directus:schema:diff": "./directus-diff.mjs",
-    "dev:directus:schema:snapshot": "dotenv -e directus.env.development directus schema snapshot > ./directus-snapshot.local.yml",
-    "dev:docker:directus:down": "docker-compose -f docker-compose.development.yml down prisma-directus-directus-dev",
-    "dev:docker:directus:up": "docker-compose -f docker-compose.development.yml up -d prisma-directus-directus-dev",
-    "dev:docker:down": "docker-compose -f docker-compose.development.yml down",
-    "dev:docker:logs": "docker-compose -f docker-compose.development.yml logs -f",
-    "dev:docker:pg:down": "docker-compose -f docker-compose.development.yml down prisma-directus-pg-dev",
-    "dev:docker:pg:up": "docker-compose -f docker-compose.development.yml up -d prisma-directus-pg-dev",
-    "dev:prisma": "dotenv -e .env.development prisma --",
+    "dev:directus:schema:snapshot": "dotenv -e ../../env/development/directus.env.development directus schema snapshot > ./directus-snapshot.local.yml",
+    "dev:docker:logs": "docker-compose -f ../../docker-compose.development.yml logs -f",
+    "dev:prisma": "dotenv -e ../../env/development/.env.development prisma --",
     "dev:prisma:datamodel": "node ./build-datamodel.mjs",
     "dev:prisma:init": "rm -rf prisma/migrations && npm run dev:prisma -- migrate dev --name init",
     "dev:reset": "npm run build && npm run dev:docker:directus:down && npm run dev:prisma:init && npm run dev:docker:directus:up",
```

Comment:
@elierotenberg I'm having an issue here with the inaccessible data, and as we discussed the other day, a reset was needed. Here as we don't have `docker-compose`, the reset is not possible do you an idea on how to fix this without using `docker-outside-of-docker` feature?